src/components/Header/NavBar/Dialog/Dialog.js

```diff
@@ -106,58 +106,42 @@ const FormDialog = memo(({ show, handleClose, address, account, amount, amountAi
 							},
 						};
 					});
-					console.log({ total: msg, total_amount });
-
+					payload = args({ msg, type: typeSubmit, fromAddress: address, totalAmount: total_amount });
 				} else {
 					total_amount = new BigNumber(data.sendAmount.toString().replaceAll(",", ""))
 						.plus(new BigNumber(total_amount))
 						.multipliedBy(consts.NUM.COSMOS_DECIMAL);
-					msg = [
-						{
-							type: typeUrl.MSG_SEND,
-							value: {
-								from_address: address,
-								to_address: data.recipientAddress,
-								amount: [
-									{
-										denom: consts.DENOM,
-										amount: handleBigNumber(data.sendAmount),
-									},
-								],
-							},
-						},
-					];
+					msg = {
+						amount: handleBigNumber(data.sendAmount),
+						denom: consts.DENOM
+					}
+					payload = { msg, type: typeSubmit, total_amount, fromAddress: address, toAddress: data.recipientAddress }
 				}
-				payload = args({ msg, type: typeSendSubmit, fromAddress: address, toAddress: data?.recipientAddress, totalAmount: total_amount.toFixed(0) });
+
 			} else if (activeTabId === 3) {
-				let executeMsg = {
-					type: typeUrl.MSG_EXECUTE_CONTRACT,
-					value: {
-						contract: config.AIRI_ADDR,
-						msg: {},
-						sender: address,
-						sent_funds: null,
-					},
-				};
+				typeSubmit = typeSend.CW20;
 				const parseTransferAiri = (address, amount) => {
-					return JSON.stringify({ transfer: { recipient: address, amount: handleBigNumber(amount) } });
+					return { transfer: { recipient: address, amount: handleBigNumber(amount) } };
 				};
-				let msgs = [];
+				let msgs = {};
 				if (multiSendData) {
 					let transferInfos = multiSendData.map(v => {
 						return { recipient: v.address, amount: handleBigNumber(v.amount) };
 					});
-					executeMsg.value.msg = JSON.stringify({ multi_transfer: { transfer_infos: transferInfos } });
+					// executeMsg.value.msg = { multi_transfer: { transfer_infos: transferInfos } };
+					msgs = { multi_transfer: { transfer_infos: transferInfos } };
 				} else {
 					total_amount = +data.sendAmount + total_amount;
```

Comment:
is the total_amount here used for the CW20 case?

Reply:
total_amount  will not be used in the new version !